Celeste.Mod.mm/Patches/OuiChapterSelectIcon.cs

```diff
@@ -18,6 +57,87 @@ public patch_OuiChapterSelectIcon(int area, MTexture front, MTexture back)
             // no-op. MonoMod ignores this - we only need this to make the compiler shut up.
         }
 
+        public extern void orig_ctor(int area, MTexture front, MTexture back);
+        [MonoModConstructor]
+        public void ctor(int area, MTexture front, MTexture back) {
+            orig_ctor(area, front, back);
+
+            // Don't bother if its not in the path, perhaps replace with an alternative check for a .meta.yaml file?
+            if (!front.AtlasPath.Contains(DetectionPath))
+                return;
```

Comment:
Is this additional subdirectory actually necessary? I'd much rather not have it necessary so you can just put animation frames into an existing directory and have it just work.